docs/source/table.rst

```diff
@@ -36,6 +36,15 @@
      - .. image:: https://licensebuttons.net/l/by-sa/4.0/80x15.png
           :target: https://creativecommons.org/licenses/by-sa/4.0
 
+   * - McGill Billboard
+     - - audio: 🔑
+       - annotations: ✅
+     - - :ref:`chords`
+       - :ref:`sections`
+     - 890
+     - .. image:: https://licensebuttons.net/l/zero/1.0/80x15.png
+          :target: http://creativecommons.org/publicdomain/zero/1.0/
```

Comment:
Am I confused or this license is not the same as the `LICENSE` attribute in the loader? in `billboard.py` it appears as [CC BY 4.0](https://creativecommons.org/licenses/by/4.0/)?

Reply:
Fixed in https://github.com/Spijkervet/mirdata/commit/30953afb30ca9bbe908c466080df15182b1ccdf9
The correct license should be a CC zero: https://ddmal.music.mcgill.ca/research/The_McGill_Billboard_Project_(Chord_Analysis_Dataset)/

Do correct me if I'm wrong - I'm not specialised in code licenses.